.travis.yml

```diff
@@ -8,8 +8,7 @@ addons:
     packages:
     - flac
 julia:
-  - 1.0
-  - 1.3
+  - 1.5
```

Comment:
You can do `1` and it'll stay on the most recent stable version, if that's what you want